bin/i18n/test/translation_status/test_translation_service.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../../../../shared/test/common_test_helper'
+require_relative '../../translation_status/translation_service'
+
+class TranslationServiceTest < Minitest::Test
+  # Use the same service for all the tests because intialization is very slow
+  @@translation_service = TranslationService.new
```

Comment:
nit: because translation_service is not shared with any subclass, making it a [class instance variables](https://www.ruby-lang.org/en/documentation/faq/8) is sufficient?

Reply:
I tried making it a class instance variable, and it looks like Minitest initializes a new instance of this class for every method run. Using a class instance variable don't persist between each test method.

Reply:
I believe what you want is a setup block; try this

```suggestion
  def setup_all
    @translation_service = TranslationService.new
  end
```

Reply:
example https://github.com/code-dot-org/code-dot-org/blob/5ab5af7d7cd988284c299a0fef314b0021b96d2c/dashboard/test/controllers/api/v1/pd/applications_controller_test.rb#L11

Reply:
I tried using that but it requires depending on `dashboard/test/test_helper` and when I try to require that I run into issue where it says it is running out of memory when trying to seed the test database or something. I don't want to spend any more time on this because there doesn't seem to be much benefit. If you want to sit down together and look at it, I'm game.

Reply:
Wow that's a lot of trouble for a small change. I'm okay with merging this code in as-is.